common/pb/moderator_commands.proto

```diff
@@ -17,3 +18,10 @@ message Command_BanFromServer {
     optional string visible_reason = 5;
     optional string clientid = 6;
 }
+
+message Command_GetBanHistory {
+    extend ModeratorCommand {
+        optional Command_GetBanHistory ext = 1001;
+    }
+    optional string user_name = 1;
```

Comment:
I think returning all bans would be reasonable.

If that's too many, consider pagination keyed by the ban's autoinc id

If that's too much work, keep the proto as `optional` so we can extend the behavior later but have the server reject the command.